### DISCUSSION TITLE
I can't use StreamingClient in API v2.

### DISCUSSION THREAD

**Discussion by legendaryangelist** [tweepy/tweepy#2126] [Questions] [answered]
**Describe the bug**
Hello.
I paid $100 and get Basic access level.
And I tried to use StreamingClient and got error like followings.

```
{"client_id":"xxxxxxxx","detail":"When authenticating requests to the Twitter API v2 endpoints, you must use keys and tokens from a Twitter developer App 
that is attached to a Project. You can create a project via the developer portal.","registration_url":"https://developer.twitter.com/en/docs/projects/overview","title":"Client Forbidden","required_enrollment":"Appropriate Level of API Access","reason":"client-not-enrolled","type":"https://api.twitter.com/2/problems/client-forbidden"}
```

But all endpoints except of StreamingClient work well.
I'm not sure why this error was happened.
Users can't use Streaming functionality in Basic level?
Or Is there any issue in API v2?

**Comment by Harmon758** [comment, marked as answer]
Streaming isn't available with Basic access.

**Reply by legendaryangelist** [reply]
Thanks for your reply
But I have one question.
I want to develop bot that replies in 1 min.
It's possible with Basic level?

**Reply by Harmon758** [reply]
You can reply to Tweets with Basic access, yes.
